### ISSUE TITLE
Find VA Form Detail Page - Release Plan

### ISSUE THREAD

**Issue by MickinSahni** [department-of-veterans-affairs/va.gov-team#11311]
## Acceptance Criteria
- [x] A completed release plan for Find VA Forms - Detail Pages

Release plan:  https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/find-va-forms-redesign/form-detail-landing-pages/release-plan.md

Related: [Batch publish Find VA Form detail pages #11155](#11155)

**Comment by jenniferlee-dsva** [comment]
@MickinSahni - Would appreciate your input. Could you take a look at the draft I link to in the ^? I'm not sure I filled everything out correctly, especially since many of the things don't quite map to CMS products.

**Comment by MickinSahni** [comment]
Hey Jen, one thought that came to mind just to completely confirm — even with the hidden launch, no index, no redirects, users will still get to the form detail pages by way of the link from search results. That jibe with your understanding? @jenniferlee-dsva

**Comment by MickinSahni** [comment]
@jenniferlee-dsva I took a look at the release plan and some structural changes. 
On the plan, am I misreading that batch publishing starts in Phase 2? 
If that's the case I'm confused about where/when UAT will happen (Phase 1)?

**Comment by jenniferlee-dsva** [comment]
This is hard to do asynchronously. Can we hash out together on call?

**Comment by jenniferlee-dsva** [comment]
@MickinSahni - answering your Qs ^. 

>even with the hidden launch (no index, no redirects), users will still get to the form detail pages by way of the link from search results. That jibe with your understanding? 

- Yes, that's right - IF we enable the form page links in the search page FE. I think we can use a feature flag to control the timing of that though. Would need Nick to confirm, but I think we could launch the form detail pages + set the feature flag so that links to the form pages from search results start showing up only once we give the thumbs up. 

>I took a look at the release plan and made some structural changes. 

- I drafted the release plan when I assumed that we wouldn't be able to do the QA review in a 'staging/dev' environment. But now that Steve can create a QA environment for form detail pages, the release plan will probably need to change.  

>On the plan, am I misreading that batch publishing starts in Phase 2? 

- I think batch publishing _would_ begin in phase 2 because we're not 'publishing' the pages during UAT (as I understand UAT). We're doing UAT first, and then, if no horrible errors are found, moving to phase 2, batch publish.  (??? But I'm not 100 on this.)

>If that's the case I'm confused about where/when UAT will happen (Phase 1)?

- TBH I'm also not sure because with Drupal content, in prod.cms, we can't test cross links (which is why Steve is creating the QA dev site for us -- to do QA in).  It seems like... 

1/ We should do QA in the QA dev environment that Steve sets up for us. 

2/ If there are major things we need to fix, we would need to fix those things in PROD (because - again, content doesn't flow up from staging to prod in Drupal; if we fix content in staging, it would get wiped out in prod and we'd have to manually re-enter in prod). 

3/ Then  - skip the template UAT and go direct to batch 1 publish. 
- Again because we can't test cross links in prod.cms, it's not possible to do traditional full UAT testing. We can only check single pages and not even links on those pages. FWIW, we've already done this level of single page UAT checking internally - both Randi and Selina validated their work on the top forms; then I validated their validation.) 

Should we still hop on a call on this? It's a lot. I'd love to target Batch 1 publish (the soft/hidden launch)  last week of August - 26, 27.

**Comment by MickinSahni** [comment]
@jenniferlee-dsva some updates ⬇️ 

> now that Steve can create a QA environment for form detail pages, the release plan will probably need to change.

yep, updated. indicated on the plan that UAT (internal + Form Managers) is happening in the QA env

>batch publishing would begin in phase 2 because we're not 'publishing' the pages during UAT

yep, we have the first pages published in prod happening as part of "phase 2" right after the form managers finish UAT in the QA env

>If there are major things we need to fix, we would need to fix those things in PROD

got it. so we'll need to keep track of any new findings from UAT and fix directly in Prod

---

We can close this out. Here are the most important dates:
* UAT - QA env: starting as early as ~8/26
* Batch 1 - Prod: as early as 8/31
* Batch 2 - Prod: as early as 9/7
* Remove no index, Implement redirects: as early as ~9/20
